api/goldens/aptos_api__tests__transactions_test__test_gas_estimation_one_block.json

```diff
@@ -0,0 +1,5 @@
+{
+  "deprioritized_gas_estimate": 1,
```

Comment:
FWIW, these labels aren't as descriptive to me as those used in the AIP. Should we consider changing them? For example: `low gas fee`, `market gas fee`, `aggressive (or priority) gas fee` seem more explanatory? 😄

Also, silly question, do we define these labels anywhere? The spec doesn't seem super descriptive: https://fullnode.devnet.aptoslabs.com/v1/spec#/operations/estimate_gas_price. Might be worth updating the spec and/or docs to call the new bands out?

Reply:
(1) Good callout about changing the spec/docs. I'll update the PR with these changes.

(2) I'm not sure what breaks if we change the json field names, so I'm playing it safe by reusing what's already there.
@gregnazario do you have any insight here? Is it possible to change the names, or would we have to create a new API endpoint and then deprecate this one?